app/src/main/java/org/oppia/android/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -176,11 +180,15 @@ class ProfileChooserFragmentPresenter @Inject constructor(
           Observer {
             if (it is AsyncResult.Success) {
               activity.startActivity(
-                (
-                  HomeActivity.createHomeActivity(
+                if (enableMultipleClassrooms.value)
+                  ClassroomListActivity.createClassroomListActivity(
                     activity,
                     model.profile.id
                   )
+                else
+                  HomeActivity.createHomeActivity(
+                    activity,
+                    model.profile.id
```

Comment:
This is not wrong but for readability I would go with:
```suggestion
                  if (enableMultipleClassrooms.value) {
        activity.startActivity(
          ClassroomListActivity.createClassroomListActivity(
            activity,
            model.profile.id
          ))
      } else {
        activity.startActivity(HomeActivity.createHomeActivity(activity, model.profile.id))
      }
```

Reply:
Done.